students/terrance_jones/lesson9/mailroom9.py

```diff
@@ -0,0 +1,177 @@
+"""
+Mailroom9.py
+Written by Terrance Jones
+Assignment for lesson 9 UW Selfpaced Online course
+"""
+
+class Donor:
+    """Creates a donor with name and and empty dontation list """
+    def __init__(self, name, donations=None):
+        self._name = name
+        if donations is None:
+            self._donations = []
+        else:
+            self._donations = [donations]
+       
+    @property
+    def name(self):
+        """Returns donor name"""
+        return self._name
+
+    @property
+    def donations(self):
+        """Returns donations list for donor"""
+        return self._donations
+
+    def append_donations(self, value):
+        self.donations.append(value)
+
+    @classmethod
+    def thank_you_letter(self, donor_name, donation_amount):
```

Comment:
the name and donations are part of your Donor object which should be used here. Something like:
```
# no decorator
def thank_you_letter(self):
    return f"Thank you {self.name} for {self.donations}"
```